extension/textext/requirements_check.py

```diff
@@ -432,14 +507,14 @@ def __init__(self, logger, config):
         self.available_tex_to_pdf_converters = {}
         self.available_pdf_to_svg_converters = {}
 
-        self.inkscape_executable_name = "inkscape"
```

Comment:
We need to have both names of programs and names of actual executables. Let's define program names in TexTextRequirementsChecker, and executable names in Windows/Linux/MacOsDefaults

Names of programs MUST match names in command line arguments. `"ghostscript" -> "--ghostscript-executable=C://.../gswin64.exe"`

First. Need to rename `*_executable_name` in TexTextRequirementsChecker to `*_name` and let them be strings as before.

Second. Let's define dictionary in Defaults which would map name to executable name list. 

linux mapping would be 

```python
class LinuxDefaults(Defaults):
    os_name = "linux"
    console_colors = "always"
    executable_basenames = {
        "inkscape": ["inkscape"],
        "python2.7": ["python2.7"],
        "pdflatex": ["pdflatex"],
        "lualatex": ["lualatex"],
        "xelatex": ["xelatex"],
        "pdf2svg": ["pdf2svg"],
        "pstoedit": ["pstoedit"],
        "ghostscript": ["ghostscript"]
    }

```